opencti-platform/opencti-graphql/src/domain/container.js

```diff
@@ -68,7 +68,17 @@ export const containersObjectsOfObject = async (context, user, { id, types, filt
   }));
   const containersObjectsRelationshipsEdges = await Promise.all(listRelationPromises);
   const containersObjectsRelationships = R.flatten(R.map((n) => n.edges, containersObjectsRelationshipsEdges));
-  const containersObjects = await Promise.all(containersObjectsRelationships.map((n) => storeLoadById(context, user, n.node.toId, n.node.toType)));
+  const containerTypedObjectsRelationships = {};
+  containersObjectsRelationships.forEach(({ node }) => {
+    if (!containerTypedObjectsRelationships[node.toType]) {
+      containerTypedObjectsRelationships[node.toType] = [];
+    }
+    if (!containerTypedObjectsRelationships[node.toType].includes(node.toId)) {
+      containerTypedObjectsRelationships[node.toType].push(node.toId);
+    }
+  });
+  const containersObjects = R.flatten(await Promise.all(Object.entries(containerTypedObjectsRelationships)
```

Comment:
Need to be discuss about how that can solve the problem.
And also the usage of await Promise.map(entries, function, { concurrency: ES_MAX_CONCURRENCY });